corehq/apps/analytics/management/commands/audit_user_in_hubspot.py

```diff
@@ -0,0 +1,89 @@
+from django.core.management import BaseCommand
+
+from corehq.apps.analytics.utils import (
+    get_blocked_hubspot_domains,
+    get_first_conversion_status_for_emails,
+    ALLOWED_CONVERSIONS,
+)
+from corehq.apps.users.models import WebUser, Invitation
+
+
+class Command(BaseCommand):
+    help = "Audit a web user's HubSpot status in HubSpot and CommCare HQ"
+
+    def add_arguments(self, parser):
+        parser.add_argument('email')
+
+    def handle(self, email, **options):
+        web_user = WebUser.get_by_username(email)
+        if not web_user:
+            self.stdout.write(self.style.ERROR(
+                "User not does not exist on CommCare HQ."
+            ))
+            return
+
+        blocked_domains = set(get_blocked_hubspot_domains())
+
+        user_domains = set(web_user.get_domains())
+        blocked_memberships = user_domains.intersection(blocked_domains)
+
+        if not blocked_memberships:
+            self.show_status_of_possibly_unblocked_user(email, blocked_domains)
+        else:
+            self.show_status_of_blocked_user(email, blocked_memberships)
+
+        self.display_first_conversion_status(email)
+
+    def show_status_of_possibly_unblocked_user(self, email, blocked_domains):
+        self.stdout.write(
+            f"\n{email} is not a member of any projects actively blocking "
+            f"data from HubSpot."
```

Comment:
Should this be on a line by itself? If a newline is not added, it should probably have a space after the period because otherwise there will be no space between the period and the next part of the message written below.
```suggestion
            f"data from HubSpot.\n"
```